health/notifications/alarm-notify.sh.in

```diff
@@ -1611,6 +1616,50 @@ send_fleep() {
     return 1
 }
 
+# -----------------------------------------------------------------------------
+# Prowl sender
+
+send_prowl() {
+    local httpcode sent=0 data message keys prio=0 alarm_url event
+    if [ "${SEND_PROWL}" = "YES" ] ; then
+        message="$(urlencode "${host} ${status_message}, \`${chart}\` (${family}), *${alarm}*\\n${info}")"
+        message="description=${message}"
+        keys="$(urlencode "$(echo "${1}" | tr ' ' ,)")"
+        keys="apikey=${keys}"
+        app="$(urlencode "application=Netdata")"
+
+        case "${status}" in
+            CRITICAL)
+                prio=2
+                ;;
+            WARNING)
+                prio=1
+                ;;
+        esac
+        pri="priority=${pri}"
+
+        alarm_url="$(urlencode ${goto_url})"
+        alarm_url="url=${alarm_url}"
+        event="$(urlencode "${host} ${status_message}")"
+        event="event=${event}"
+
+        data="${keys}&${pri}&${alarm_url}&${application}&${event}&${message}"
```

Comment:
Replace `application` with `app`

Reply:
Prowl API documentation doesn't agree with you: https://www.prowlapp.com/api.php#add :stuck_out_tongue: